Arm backend: Enable int16 rsqrt on Ethos-U55/U85

3l1 opened the pull request:
Summary:
Fix Rsqrt op for int16
Add unit tests

Differential Revision: D83802158




cc @freddan80 @per @zingo @oscarandersson8218 @digantdesai

3l1 commented:
@pytorchbot label "release notes: none"

digantdesai reviewed (approved):
Enable just the tests?

3l1 committed:
Enable int16 rsqrt on Ethos-U55/U85 (#14770)

Summary:

Fix Rsqrt op for int16
Add unit tests

bypass-github-export-checks
bypass-github-pytorch-ci-checks
bypass-github-executorch-ci-checks

Reviewed By: Ninja91, digantdesai

Differential Revision: D83802158

zingo committed:
Merge branch 'main' into export-D83802158

zingo commented:
I see this errors
[pull / unittest-arm-backend-with-no-fvp (test_pytest_ops) / linux-job (pull_request)](https://github.com/pytorch/executorch/actions/runs/19127321633/job/54660017256?pr=14770)
```
FAILED backends/arm/test/ops/test_rsqrt.py::test_rsqrt_int16_tosa_INT[rand_4d_1]
FAILED backends/arm/test/ops/test_rsqrt.py::test_rsqrt_int16_tosa_INT[rand_4d_2]
```
@3l1 could they be caused by the PR?
